services/spar/test-integration/Test/Spar/DataSpec.hs

```diff
@@ -195,7 +195,7 @@ spec = do
 
 
 testSPStoreID
-  :: forall m a. (m ~ ReaderT Data.Env (ExceptT TTLError Client), Typeable a)
+  :: forall m (a :: *). (m ~ ReaderT Data.Env (ExceptT TTLError Client), Typeable a)
```

Comment:
Now that PolyKinds is enabled this function is SO polymorphic it couldn't even tell which kind it wants; I just went with *.